test/run_pass/test_ble_setservices_central.js

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2016-present Samsung Electronics Co., Ltd. and other contributors
```

Comment:
@yichoi Yes, you're right. Starting year is 2015. It was my mistake. I see. You want to keep the first number as the created year. Although I am not agree with that, I will follow.